schemas/endpoints/partitionLoad.yaml

```diff
@@ -0,0 +1,103 @@
+PartitionLoadEndpoint:
+  get:
+    summary: Get the resource load for each partition
```

Comment:
to be consistent across all endpoint, can we add a `operationId` here(like in `rebalance.yaml`)?